### PR TITLE
7.0.0 - Support and Puppet 4.0

### PR DESCRIPTION
**Pull request by ffquintella**
This code runs on puppet 4.0 and Puppet Enterprise and has support for Jira 7.0.0

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -103,11 +113,11 @@
 
   if $jira::db == 'mysql' and $jira::mysql_connector_manage {
     if $jira::staging_or_deploy == 'staging' {
-      class { '::jira::mysql_connector':
+      class { 'jira::mysql_connector':
         require => Staging::Extract[$file],
       }
     } elsif $jira::staging_or_deploy == 'deploy' {
-      class { '::jira::mysql_connector':
+      class { 'jira::mysql_connector':
```
Comment (bastelfreak) [inline manifests/install.pp]: why don't you want to use the top level scope?

File: manifests/install.pp [hunk 2]
```diff
@@ -40,10 +40,20 @@
     }
   }
 
-  $file = "atlassian-${jira::product}-${jira::version}.${jira::format}"
+  # https://www.atlassian.com/software/jira/downloads/binary/atlassian-jira-software-7.0.0-jira-7.0.0-x64.bin
+
+  $arr_version = split($jira::version, '.')
+
+  $major = 0 + $arr_version[0]
```
Comment (bastelfreak) [inline manifests/install.pp]: Why you add 0? An interesting method to cast something to an int?

File: manifests/install.pp (hunk 1 above)
Comment (ffquintella) [inline manifests/install.pp]: Not needed  and it creates problems with puppet 4.0 and lint

File: manifests/install.pp (hunk 2 above)
Comment (ffquintella) [inline manifests/install.pp]: This is a way to convert the string into a number. But this isn't  the last code. 

Int the end I just used $jira::version[0] 

Let me see why it came to you

**Comment by ffquintella** [comment]
Sorry the last code had problems on the merge to the master this one should be the latest version

**Comment by ffquintella** [comment]
This pull is wrong I missed up the merge.I will redo it

**Pull request closed without merging**